pywbem_mock/config.py

```diff
@@ -0,0 +1,60 @@
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+#
+"""
+Pywbem_mock supports some configuration variables provide configuration and
+behavior control of the mock environment
+
+These configuration variables can be modified by the user directly after
+importing pywbem. For example:
+
+::
+
+    import pywbem_mock
+    pywbem_mock.config.ENFORCE_INTEGER_RANGE = False
+
+Note that the pywbem source file defining these variables should not be changed
+by the user. Instead, the technique shown in the example above should be used to
+modify the configuration variables.
+"""
+
+# This module is meant to be safe for 'import *'.
+
+__all__ = ['OBJECTMANAGERNAME', 'SYSTEMNAME',
+           'SYSTEMCREATIONCLASSNAME', 'DEFAULT_MAX_OBJECT_COUNT',
+           'OPEN_MAX_TIMEOUT']
+
+#: Name for the ObjectManager if the CIM_ObjectManager class is implemented.
+
+OBJECTMANAGERNAME = 'MyFakeObjectManager'
+
+#: The name of the mock system. This string value becomes part of the
+#: CIM_ObjectNamager instance and CIM_Namespace instances
+
+SYSTEMNAME = 'Mock_Test_WBEMServerTest'
```

Comment:
Can we use a less test oriented value for SYSTEMNAME, e.g. "MockSystem"?

Reply:
DONE